chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/CertifiedProductDTO.java

```diff
@@ -48,7 +49,11 @@
     private String productAdditionalSoftware;
     private Long pendingCertifiedProductId;
     private Boolean transparencyAttestation = null;
-    private Integer rwtEligiblityYear;
+    private String rwtPlanUrl;
+    private Date rwtPlanSubmissionDate;
```

Comment:
Do the new date fields needs getters/setters using Util.getNewDate?

Reply:
Same as above.